devices/tuya.js

```diff
@@ -1625,6 +1625,35 @@ module.exports = [
                 .withDescription('Alarm temperature status'),
         ],
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_nnrfa68v'}],
+        model: 'E6',
+        vendor: 'Nous',
```

Comment:
Vendor name should match the filename, please move this to a new file `devices/nous.js`

Reply:
Done.